internal/storage/utilities.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
+
+package storage
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// stub this so we can make unit tests for depending packages

```

Comment:
I would be a bit more explicit here, something like "we export this as a variable here so that it can be mocked from other packages to help with testing"